ext/gd/tests/similarity.inc

```diff
@@ -7,15 +7,8 @@
 
 /**
  * Gets the individual components of an RGB value.
- *
- * @param int $color
- * @param int $red
- * @param int $green
- * @param int $blue
- *
- * @return void
  */
-function get_rgb($color, &$red, &$green, &$blue)
+function get_rgb(int $color, &$red, &$green, &$blue): void
```

Comment:
Are the by-ref args out params? It might still make sense to have doc comments for them